.github/workflows/tests.yaml

```diff
@@ -0,0 +1,41 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master, v5 ]
+  pull_request:
+    branches: [ master, v5 ]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  ansible:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Ansible
+        run: |
+          python -m pip install --upgrade wheel
```

Comment:
nit: I don't think it's actually needed since you don't plan to build your own wheels. If I'm not mistaken, `pip` installs wheels out of box. It's more common to upgrade `pip` itself as the first step (I dunno why).

Reply:
I've copied it from another workflow :) I don't remember _why_ I needed to do that there, though... So maybe it's not needed indeed. It's a good point about updating `pip`. Let me do that instead!